include/magma/vk/pipeline/PipelineInfo.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+#include <vulkan/vulkan.hpp>
+#include <vector>
+
+class PipelineLayoutInfo {
+public:
+    PipelineLayoutInfo();
+    PipelineLayoutInfo(const std::vector<vk::DescriptorSetLayout> &descriptorSetLayouts);
+    PipelineLayoutInfo(const vk::DescriptorSetLayout &descriptorSetLayout);
+
+    ~PipelineLayoutInfo() = default;
+
+    const vk::PipelineLayoutCreateInfo &getPipelineLayoutInfo() const { return _pipelineLayoutInfo; }
```

Comment:
That's the only getter in this class (and no more getters are supposed to be there).
So it's name can be more simple, like get() or getInfo().